### ISSUE TITLE
loss is increased rapidly during training on Step2. Is EfficientDet unstable?

### ISSUE THREAD

**Issue by StephenGreat** [xuannianz/EfficientDet#14]
I have used my own dataset to train EfficientDet.
My params:
phi=0
gpu=0
workers=1
Other params are default following Step1 and Step2.
During training Step1, the loss is decreased stably. But the loss is increased rapidly on Step2.
The loss is increased from 0.5 to 57!
So is EfficientDet unstable? Or are there some tricks of training.
